llvm/docs/Passes.rst

```diff
@@ -543,6 +543,15 @@ variables with initializers are marked as internal.
 An interprocedural variant of :ref:`Sparse Conditional Constant Propagation
 <passes-sccp>`.
 
+``ir-normalizer``: Transforms IR into a canonical form that's easier to diff
+--------------------------------------------------------
+
+This pass aims to transform LLVM Modules into a canonical form by reordering and
+renaming instructions while preserving the same semantics. The canonicalizer makes
+it easier to spot semantic differences while diffing two modules which have undergone
+two different passes.
+
+-----------------------------------
```

Comment:
Shouldn't this line be removed? and the line underscoring the "``ir-normalizer``: Transforms IR into a canonical form that's easier to diff" be made a bit longer?